packages/cli/package.json

```diff
@@ -16,7 +16,6 @@
     "directory": "packages/cli"
   },
   "scripts": {
-    "prepare": "pnpm build",
```

Comment:
remove all prepare scripts for `yarn build`

Reply:
we could prob just do this ahead of nx changes, just to make install experience better (noting that folks will have to manually run `pnpm build` after if they need built files)

Reply:
I think in general the strategy is to keep this pr a wip and pull out tiny safe prs from it into seperate prs

Reply:
extracted it here: https://github.com/latticexyz/mud/pull/623